### DISCUSSION TITLE
Mark environment secrets as non-sensitive

### DISCUSSION THREAD

**Discussion by foxted** [community/community#35566] [Actions]
When defining an environment secret in a repository, like `PROJECT_ID = project` and the environment URL happen to include that string (for example: https://project.com), the output will be masked in the actions summary:

![Screen Shot 2022-10-08 at 19 06 51](https://user-images.githubusercontent.com/1385160/194734251-d4c52b85-9d3d-4403-87a1-768214dfe9c9.png)

Is there a way to get around that? Would there be a way to mark an environment secret as "non-sensitive"?

**Comment by juliandunn** [comment]
[It's on the roadmap](https://github.com/github/roadmap/issues/575)! Code complete and assuming there are no last minute impediments, we hope to have this by the end of the year.
